cli/lib/tasks/verify.ts

```diff
@@ -1,29 +1,38 @@
-const _ = require('lodash')
-const chalk = require('chalk')
-const { Listr } = require('listr2')
-const debug = require('debug')('cypress:cli')
-const { stripIndent } = require('common-tags')
-const Promise = require('bluebird')
-const logSymbols = require('log-symbols')
-const path = require('path')
-const os = require('os')
-
-const verbose = require('../VerboseRenderer')
-const { throwFormErrorText, errors } = require('../errors')
-const util = require('../util')
-const logger = require('../logger')
-const xvfb = require('../exec/xvfb')
-const state = require('./state')
-
-const VERIFY_TEST_RUNNER_TIMEOUT_MS = +util.getEnv('CYPRESS_VERIFY_TIMEOUT') || 30000
-
-const checkExecutable = (binaryDir) => {
+import _ from 'lodash'
+import chalk from 'chalk'
+import { Listr } from 'listr2'
+import Debug from 'debug'
+import { stripIndent } from 'common-tags'
+import Bluebird from 'bluebird'
+import logSymbols from 'log-symbols'
+import path from 'path'
+import os from 'os'
+import verbose from '../VerboseRenderer'
+import { throwFormErrorText, errors } from '../errors'
+import util from '../util'
+import logger from '../logger'
+import xvfb from '../exec/xvfb'
+import state from './state'
+
+const debug = Debug('cypress:cli')
+
+const VERIFY_TEST_RUNNER_TIMEOUT_MS = (() => {
+  const verifyTimeout = +(util?.getEnv('CYPRESS_VERIFY_TIMEOUT') || 'NaN')
+
+  if (_.isNumber(verifyTimeout) && !_.isNaN(verifyTimeout)) {
+    return verifyTimeout
+  }
+
+  return 30000
+})()
```

Comment:
@AtofStryker This seems a bit overly complex :/

Reply:
it's technically correct because `CYPRESS_VERIFY_TIMEOUT` could be undefined, though there shouldn't be a common case where that happens. This more so guarantees that we always get the default if we can't resolve  `CYPRESS_VERIFY_TIMEOUT` so it's a bit more defensive which isn't necessarily a bad thing